### DISCUSSION TITLE
"Error on DumpDDLs: pq: database ... does not exist" with unix sockets

### DISCUSSION THREAD

**Discussion by nyarly** [sqldef/sqldef#637] [Feature requests]
## Platform

* OS: Linux 
* RDBMS: PostgreSQL
* Version: 0.17.17

## --export output

```shell
> psqldef -h (pwd)/devsupport/db_sockets --export mydatabase
2024/09/16 16:55:16 Error on DumpDDLs: pq: database "home/nyarly/dev/myproject/devsupport/db_sockets/mydatabas" does not exist
```
(note: missing leading `/` and trailing `e`)

## Input SQL

(long DB schema - I'm porting an existing app)

## Current output

```
2024/09/16 16:55:16 Error on DumpDDLs: pq: database "home/nyarly/dev/myproject/devsupport/db_sockets/mydatabas" does not exist
```

## Expected output

I mean, not that. Ideally migrations.

**Comment by gfx** [comment]
Fixed in https://github.com/sqldef/sqldef/pull/992
